modules/shared/components/organisms/PostCreation/PostCreation.tsx

```diff
@@ -159,6 +161,7 @@ const PostCreation: FC<IPostCreation.IProps> = ({
         await logoutUser();
         redirectToLoginPage();
       }
+      router.reload();
```

Comment:
use 
```typescript
router.replace(router.asPath) as Promise<NextRouter>
```
as if the user tried to go back to the previous page, next js will ignore the browser refresh and that's what he expects